recipes/recastnavigation/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+#include "Recast.h"
+
+int main() {
+    std::cout << (rcMin(10, 20) == 10);
```

Comment:
I haven't checked but are you 100% sure that `rcMin` is a library function? I'm asking because minimum is a really simple function.
If it's defined and implemented in a header, we might not actually check the libraries.
We want to check code in the packaged libraries.